www/src/pages/index.tsx

```diff
@@ -1,45 +1,16 @@
-import clsx from "clsx";
-import Link from "@docusaurus/Link";
-import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
-import Layout from "@theme/Layout";
-import HomepageFeatures from "@site/src/components/HomepageFeatures";
-import Heading from "@theme/Heading";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import { Route } from "react-router-dom";
 
-import styles from "./index.module.css";
-
-function HomepageHeader() {
-  const { siteConfig } = useDocusaurusContext();
-  return (
-    <header className={clsx("hero hero--primary", styles.heroBanner)}>
-      <div className="container">
-        <Heading as="h1" className="hero__title">
-          {siteConfig.title}
-        </Heading>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/introduction"
-          >
-            Skip Overview
-          </Link>
-        </div>
-      </div>
-    </header>
-  );
-}
-
-export default function Home(): JSX.Element {
-  const { siteConfig } = useDocusaurusContext();
-  return (
-    <Layout
-      // title={`Hello from ${siteConfig.title}`}
-      description="Description will go into a meta tag in <head />"
-    >
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
-      </main>
-    </Layout>
-  );
-}
+export default () => (
+  <BrowserOnly>
+    {() => {
+      <Route
+        path="/"
+        component={() => {
+          window.location.href = "https://skiplabs.io/";
```

Comment:
This is the key change for the bug we noticed on Friday, right?  i.e. it'll act like a regular non-SPA link to the home page, jumping outside of docusaurus

Reply:
AFAIU, yes, to get an actual redirect, that does a genuine http fetch, without being hijacked by whatever this SPA business is, one has to resort to something like this that just assigns to a global, but doubly-delayed by both being in a closure and under a BrowserOnly tag in order to hide it well enough that docusaurus can't break it.